### DISCUSSION TITLE
Communicate breaking changes appropriately in release notes

### DISCUSSION THREAD

**Discussion by vkz** [Chia-Network/chia-blockchain#5682] [Ideas]
With 1.1.6 IIUC you effectively broke the *implicit* API most everyone not running GUI relied upon i.e. log messages. Like it or not we've all been tracking connectivity, farmer and harvester status etc by grepping INFO level log messages. Therefore the moment you start messing with your logs is the moment you've broken your external API. I'm not about to ask you to bump appropriate minor version in semver _just for that_ but please consider communicating such instances very clearly and in detail in your release notes. 

Case in point. `farmer_info` which you may use to track farmer receiving responses to challenges from remote harvester is no longer there.

This is far from sufficient:
>Many safe warnings have been changed to DEBUG, and the INFO level log has been simplified and made smaller

Thank you

**Comment by lvcivs** [comment]
You can't rely on log messages to build up any sort of functionality on top. Logging can and will change between releases. Calling it an "implicit API" is just wrong imho.

**Reply by vkz** [reply]
Thank you for stating the obvious and educating me. 
I've not chosen to rely on logs, rather I have been forced into it by lack of other means. 
Apparently and I quote:
> This is what success looks like

**Comment by vkz** [comment]
Seems not everyone agrees, so let me just remind that (a) using detailed INFO (that's effectively now DEBUG) level and logs has been an official party line since day one - recommended to users both on Keybase and on wiki and (b) Wiki not only sanctions parsing logs but actually recommends it e.g. that `farming_info` trick is straight out of Wiki.

To reiterate. Noone chooses log messages to be their API and yet here we are.

**Comment by ghost** [comment]
So this would to be a pretty inadequate documentation of a pretty big change in INFO logs. Is there a suggestion from the Chia team of how to adequately monitor the communication between farmer and harvester. It also would seem that you would want to also update the documentation on the wiki with such a large change.
